sync/class.jetpack-sync-module-comments.php

```diff
@@ -11,6 +11,9 @@ public function init_listeners( $callable ) {
 		add_action( 'deleted_comment', $callable, 10 );
 		add_action( 'trashed_comment', $callable, 10 );
 		add_action( 'spammed_comment', $callable, 10 );
+		add_action( 'jetpack_sync_trashed_post_comments', $callable, 10 );
+
+		add_action( 'trashed_post_comments', array( $this, 'trashed_post_comments_handler' ), 10, 2 );
```

Comment:
why not just sync the trashed_post_comments action directly, rather than creating a new action?